portfolio/src/main/java/com/google/sps/servlets/PollDataServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/poll-data")
+public class PollDataServlet extends HttpServlet {
+
+  private LinkedHashMap<Integer, Integer> bigfootSightings = new LinkedHashMap<>();
```

Comment:
would it make sense to store this as a json string instead rather than de-serialize it every time a GET request is made?